builds/any/rootfs/stretch/common/overlay/etc/systemd/system/switchdev-online@.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Online state for switchdev device %i
+ 
+# See e.g.
+# https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
+Wants=network-pre.target
+Before=network-pre.target
```

Comment:
I am wondering about this line, as the drop-in above has

    After=network-pre.target

Isn’t that a contradiction?

/cc @tjjh89017

Reply:
@paulmenzel 
Suppose you mean `After=network-pre.target` in `networking.service.d/switchdev-online.conf` conflicts with `Before=network-pre.target` in `switchdev-online@.service`

In Rothcar's patch, he want to guarantee that `networking.service` will always be executed after ONLP insert ASIC driver.
he made `switchdev-online@.service` will always be before `network-pre.target`.
and networking.service (ifupdown2) will always be after `network-pre.target`

So you can see this order below in Rothcar's patch:
1. execute switchdev-online@.service (load driver, let Linux can see those port)
2. reach network-pre.target
3. execute networking.service (load config. if ports isn't existing, it will fail)

and in my PR #65 
try to guarantee this order below (from No. 3 to 5)
1. execute switchdev-online@.service (load driver, let Linux can see those port)
2. reach network-pre.target
3. execute networking.service (load config. if ports and config are mismatch, it will fail)
4. reach network.target
5. execute other service which need network (e.g. isc-dhcp-server)

Reply:
Sorry, I totally screwed up, and in `networking.service.d/switchdev-online.conf` only read `switchdev-online` and associated it with the `switchdev-online` unit. But it belongs to to the unit `networking.service`. Sorry about that.